src/linagora.esn.calendar/app/planning/calendar-planning.less

```diff
@@ -47,4 +47,23 @@
   .fc-list-item-title {
     font-weight: 500;
   }
-}
\ No newline at end of file
+
+  .planning-event-accepted {
```

Comment:
why an empty class ?

Reply:
@asmaNode Well, the code responsible for styling the event on main calendar also having an empty class for accepting event. So I just want to follow that since it have been merged

Reply:
We should remove it if it is not necessary, IMO